frameworks/PHP/symfony/deploy/nginx.conf

```diff
@@ -41,30 +41,6 @@ http {
         listen       8080;
         server_name  localhost;
 
-        #charset koi8-r;
-
-        #access_log  logs/host.access.log  main;
-
-        #location / {
-        #    root   html;
-        #    index  index.html index.htm;
-        #}
-
-        #error_page  404              /404.html;
-
-        # redirect server error pages to the static page /50x.html
-        #
-        #error_page   500 502 503 504  /50x.html;
-        #location = /50x.html {
-        #    root   html;
-        #}
-
-        # proxy the PHP scripts to Apache listening on 127.0.0.1:80
-        #
-        #location ~ \.php$ {
-        #    proxy_pass   http://127.0.0.1;
-        #}
-
         root /home/tfb/FrameworkBenchmarks/php-symfony/public/;
```

Comment:
You also need to modify here to 
``/home/vagrant/FrameworkBenchmarks/frameworks/PHP/php-symfony/public/;``
To follow the following modif in setup.sh:
``sed -i 's|root .*/FrameworkBenchmarks/frameworks/PHP/php-symfony|root '"${TROOT}"'|g' deploy/nginx.conf``